src/ray/core_worker/core_worker.cc

```diff
@@ -990,6 +1010,17 @@ void CoreWorker::HandleGetObjectStatus(const rpc::GetObjectStatusRequest &reques
   }
 }
 
+void CoreWorker::HandleNotifyActorCreated(const rpc::NotifyActorCreatedRequest &request,
```

Comment:
This should be done on reply to the actor creation task instead of a separate RPC.

Reply:
done